diff --git a/node_modules/@brooons/react-native-bluetooth-escpos-printer/android/build.gradle b/node_modules/@brooons/react-native-bluetooth-escpos-printer/android/build.gradle
index def666c..ef23882 100644
--- a/node_modules/@brooons/react-native-bluetooth-escpos-printer/android/build.gradle
+++ b/node_modules/@brooons/react-native-bluetooth-escpos-printer/android/build.gradle
@@ -20,12 +20,12 @@ buildscript {
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 28
-    buildToolsVersion "28.0.3"
+    compileSdkVersion 34
+    buildToolsVersion "34.0.0"
 
     defaultConfig {
         minSdkVersion 16
-        targetSdkVersion 24
+        targetSdkVersion 34
         versionCode 1
         versionName "1.0"
     }
